### PR TITLE
Remove a related project (no longer available, domain on sale)

### PR DESCRIPTION
**Pull request by katzien**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -78,4 +77,4 @@ We Are Working On
 * The aggregation functionality is ignored completely
 * The code is... a mess. Deadlines do that to you, we're working on it
 * The default table schema isn't indexed all the places it needs to be
-* Easier ways to diff URLs
\ No newline at end of file
+* Easier ways to diff URLs
```
Comment (aik099) [inline README.markdown]: What was changed here?

Comment (katzien) [reply README.markdown]: Oops, no idea! I'm guessing it was just whitespace, I'll try and revert this bit.

Comment (katzien) [reply README.markdown]: Dang it, there's an extra new line at the end of the file but it won't let me remove it. I'll just create a new PR and hopefully it won't have the newline this time.

Comment (aik099) [reply README.markdown]: You could have edited this one as well.

Comment (katzien) [reply README.markdown]: Yeah, I tried editing (twice) and it was still the same, so I thought I'll just try again from scratch. Still no luck as it turned out :) Sorry for the duplicates!

**Pull request closed without merging**